src/Core/WinSWCore/XMLSchema.xsd

```diff
@@ -0,0 +1,140 @@
+﻿<?xml version="1.0"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <xs:element name="service">
```

Comment:
It would be great to add some documentation to the fields so that it can be displayed in the XML editor like VS Code

Reply:
> @oleg-nenashev What sort of details should be added in xmlConfigFile.md? I think adding whole schema file will be an overkill. Will just specifying that we are using embedded XSD schema file to validate XML config file and user can follow that schema file be enough ?

@buddhikac96 adding a reference to the schema and briefly explaining how to use it?

Reply:
Would be great! I will.

Reply:
OK, will wait for it before merging